docs/kms_rest_cli.md

```diff
@@ -29,9 +29,11 @@ Parameters can be set by command line arguments or environment variables:
     --kms-secrets-database-url string         The URL of the database for KMS secrets. Not needed if using in-memory storage. For CouchDB, include the username:password@ text if required. Alternatively, this can be set with the following environment variable: KMS_SECRETS_DATABASE_URL
     --kms-secrets-database-prefix string      An optional prefix to be used when creating and retrieving the underlying KMS secrets database. Alternatively, this can be set with the following environment variable: KMS_SECRETS_DATABASE_PREFIX
 
-    --operational-kms-storage-type string     The type of storage to use for Operational (user-specific) KMS. Supported options: mem, couchdb, sds. Alternatively, this can be set with the following environment variable: KMS_OPERATIONAL_KMS_STORAGE_TYPE
-    --operational-kms-storage-url string      The URL of storage for Operational KMS. Not needed if using in-memory storage. For CouchDB, include the username:password@ text if required. Alternatively, this can be set with the following environment variable: KMS_OPERATIONAL_KMS_STORAGE_URL
-    --operational-kms-storage-prefix string   An optional prefix to be used when creating and retrieving the underlying Operational KMS storage. Alternatively, this can be set with the following environment variable: KMS_OPERATIONAL_KMS_STORAGE_PREFIX
+    --kms-secrets-master-key-path string      The path to the file with master key to be used for secret lock. If missing noop service lock is used. Alternatively, this can be set with the following environment variable: KMS_SECRETS_MASTER_KEY_PATH
```

Comment:
let's use a different term than "master"

Reply:
main?

Reply:
but it will lead to discrepancy with the framework, i.e. framework still uses "master" terminology